### ISSUE TITLE
RBS late-2019 GTX booting Ubuntu 19 from Live USB

### ISSUE THREAD

**Issue by dror-g** [rolandguelle/razer-blade-stealth-linux#39]
Razer Blade Stealth Late 2019 GTX 1650 model
Out of the box - managed to boot Mint 19.1 (Xenial) from USB.
Did not install.
After some windows use, Windows & Nvidia updates -
Will not boot from USB.

Tried Ubuntu 19.10/19.4, Mint 18.4/19.1/19.2.
Tried nomodeset, nouveau.modeset=0 (or 1), 
Fast Boot on and off
Secure Boot on and off - This actually changes consule resolution, with Mint showing blank screen without it.

Errors presented:
`sda access beyond end of device`
and
`unknown input` (this one is non-halting though)

Edit: Also seeing -
`thermal thermal_zone1: critical temprature reached (198 C), shutting down`
I get that on ubuntu, always 198c.
Tried boot parameters like `thermal.act=-1` , `thermal.off=1`, `thermal.noact=1` - no effect. 

Can someone confirm?

**Comment by dror-g** [comment]
I can boot Ubuntu 18.04.3 from USB created from Win with Rufus in dd mode,
Install stalled, failed to mount the EFI partition to /target/boot, 
Had to quit installer, use fdisk /dev/nvme0n1 and delete all partitions,
Restarted installer and it worked.

Updated to kernel 5.3.8 from teejee2008 PPA - 
Nvidia propriety driver & wifi now work,
Bluetooth still broken, and suspend loop as described elsewhere.

So I guess this issue is specific to Ubuntu 19.

**Comment by danielbodart** [comment]
I booted my Razor Stealth Late 2019 from the normal 19.10 USB (Created with Etcher)

In BIOS I have Secure Boot Turned off (but it did work with it on as well)

Then instead of pressing F12 which seemed to only sometimes work, I pressed DEL and then from the last option in the bios chose to boot from the UEFI USB directly.

I then ignored setting any grub options and did Try Ubuntu. From there I was able to then click the Install Icon from the Ubuntu desktop.

NVidia worked out of the box as did the Wifi. Only issue I had was the suspend loop bug
